xoto3/dynamodb/write_versioned/specify.py

```diff
@@ -60,7 +65,12 @@ def presume(
             tables={
                 **transaction.tables,
                 table_name: _TableData(
-                    items={**table_data.items, hkey: item_value},
+                    items={
+                        **table_data.items,
+                        hkey: dynamodb_prewrite(item_value, prewrite_transform)
+                        if item_value
+                        else item_value,
```

Comment:
Is this to preserve a variety of possible falsey values in `item_value` or could it be simplified to `else None`?

Reply:
you're right, `None` would be more readable here.